README.md

```diff
@@ -199,6 +199,38 @@ and don't use `Marker` composables (unless you don't care about `onClick`
 events). Clustering is the only use-case tested with `MapEffect`, there may be
 gotchas depending on what features you use in the utility library.
 
+## Components
```

Comment:
I'm open to better suggestions on what to call these. Widgets? Composables?

Reply:
I kinda like "Widgets" as it helps emphasize that it is built separate from the core Maps SDK. Wdyt?